test/controllers/home_controller_test.rb

```diff
@@ -1,8 +1,24 @@
 require 'test_helper'
 
 class HomeControllerTest < ActionController::TestCase
+  setup do
+    @user = users(:one)
+    sign_in @user
+
+    @task = tasks(:one)
+  end
+
   test "should get index" do
     get :index
-    assert_response :success
+    # TODO - Fix?
```

Comment:
Well, if the user is signed in, per your `setup` block, they won't land on the `index` page. They'll be redirected somewhere. Maybe you could just sign them in as part of the dashboard test?